crates/runtime/src/metrics_server/mod.rs

```diff
@@ -64,31 +68,58 @@ where
         let stream = match listener.accept().await {
             Ok((stream, _)) => stream,
             Err(e) => {
-                tracing::warn!(
+                tracing::debug!(
```

Comment:
**question:** Should this log be `debug!`?

Reply:
I don't think so. It was fairly noisy when I tried connecting incorrectly (I.e HTTP instead of HTTPS). Axum serve didn't report them so I opted to hide it behind a debug